elements/pfe-band/demo/index.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>PatternFly Element: Band Demo</title>
+
+    <noscript>
+      <link href="../../pfelement/pfelement-noscript.min.css" rel="stylesheet">
+    </noscript>
+
+    <link href="../../pfelement/pfelement.min.css" rel="stylesheet">
+    <link href="../../pfe-layouts/pfe-layouts.css" rel="stylesheet">
+
+    <!-- uncomment the es5-adapter if you're using the umd version -->
+    <script src="/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>
+    <script src="/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.5/require.min.js"></script>
+    <script>require([
+      '../pfe-band.umd.js',
+      '../../pfe-card/pfe-card.umd.js',
+      '../../pfe-cta/pfe-cta.umd.js'
+    ])</script>
+
+    <style>
+    body {
+      margin: 0;
+    }
+    /* Example CSS variable overrides
+    :root  {
+      --pfe-theme--container-spacer: 4px;
+    }
+
+    pfe-band {
+      --pfe-band--padding-vertical: 1px;
+      --pfe-band--padding-horizontal: 1px;
+      --pfe-band_aside--layout: 380px;
+    }
+    */
+
+    /* Set typography styles using a class */
+    .custom-band-summary {
+        font-size: 20px;
+        margin: 0;
+    }
+    </style>
+  </head>
+  <body unresolved>
+    <h1>&lt;pfe-band&gt;</h1>
+    <pfe-band pfe-color="lightest" pfe-aside-desktop="right" pfe-aside-mobile="bottom" pfe-aside-height="body">
+      <header slot="pfe-band-header">
```

Comment:
@castastrophe hmm, the band headers and footers are not rendering http://localhost:1234/elements/pfe-band/demo/

Reply:
They are now, I'm not sure what the issue was here but let me know if this isn't resolve now.